Revit_Engine/Create/Requests/WorksetRequest.cs

```diff
@@ -0,0 +1,48 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.ComponentModel;
+
+using BH.oM.Adapters.Revit.Enums;
+using BH.oM.Adapters.Revit;
+using BH.oM.Base;
+using BH.oM.Reflection.Attributes;
+
+namespace BH.Engine.Adapters.Revit
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Creates an IRequest that filters all elements in a given workset.")]
+        [Input("worksetName", "Workset Name")]
+        [Output("WorksetRequest")]
+        public static WorksetRequest WorksetRequest(string worksetName)
```

Comment:
By the way, just to let you know if you didn't know:
If the Create method is "elementary" like this, we now have the automatic `Create` generator. So if your type is simple, you will automatically find in the UI the create method for it, without having to write an explicit Create.
It basically auto-generates a constructor that takes as an input all the public `{get;set;}` properties and populates them.

I dont think we haven't yet formulated what will happen to the "handwritten" Create method in the future. I heard Al mentioning that we will remove all of them to leave only the automatically generated one. 
However I personally think it's still good practice to write them as you did. So leave them in! :)

Reply:
Yeah, I have heard so too, and yeah, I do not know the exact plan too 🙊 So decided to add them, which also helped me to get my head around the problem better.

Reply:
Would probably agree with @alelom s first comment. These wont be needed and are to be deprecated and then removed globally across all engines going forward, so would not add in new ones. Better then to move the descriptions to the object itself. @al-fisher

Reply:
Yes please! I would indeed move descriptions across to oM and remove any Creates with only defining primary BHoM properties as arguments. 
The object initializer syntax for any standard BHoM object with fully public get sets is now auto exposed in UI. Hope that is OK?

Reply:
Oke, I added descriptions to classes and deleted irrelevant Create methods.